Makefile

```diff
@@ -52,7 +52,7 @@ fmt:
 	$(CRI_BIN) run --rm \
 	           --volume `pwd`:$(CURDIR):Z \
 	           --workdir $(CURDIR) \
-	           $(GO_IMAGE_NAME):$(GO_IMAGE_TAG) gofmt -w .
+	           $(GO_IMAGE_NAME):$(GO_IMAGE_TAG) gofmt -w -s .
```

Comment:
Could you please add an explanation to the commit message about what this flag does and why is it needed?

Reply:
when I did, I realized I don't want this flag. will remove it from here and from linter